inference-engine/tests/unit/frontends/onnx_import/CMakeLists.txt

```diff
@@ -2,20 +2,20 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-if (NOT NGRAPH_USE_PROTOBUF_LITE)
-    set(TARGET_NAME onnxImporterUnitTests)
+set(TARGET_NAME onnxImporterUnitTests)
 
-    addIeTargetTest(
-            NAME ${TARGET_NAME}
-            ROOT ${CMAKE_CURRENT_SOURCE_DIR}
-            LINK_LIBRARIES
-                gtest
-                gtest_main
-                onnx_importer
-            DEFINES
-                ONNX_MODELS_DIR=\"${CMAKE_CURRENT_SOURCE_DIR}/models\"
-            ADD_CPPLINT
-            LABELS
-                ONNX
-    )
-endif()
+addIeTargetTest(
+        NAME ${TARGET_NAME}
+        ROOT ${CMAKE_CURRENT_SOURCE_DIR}
+        LINK_LIBRARIES
+            gtest
+            gtest_main
+            onnx_importer
+        DEFINES
+            ONNX_MODELS_DIR=\"${CMAKE_CURRENT_BINARY_DIR}/models\"
```

Comment:
On Azure, build directory will be removed after install. So, generated models shall be installed as 'tests' and 'unit-test' shall search this location

Reply:
Fixed. Such tests now finds their models relatively to test binary in `./test_model_zoo` in both install and after build scenarios.